src/observability/publisher.go

```diff
@@ -20,19 +20,19 @@ func ProcessSystemSummary() {
 	tempSummarizerMap := common.MoveMap(SummarizerMap)
 	SummarizerMapMutex.Unlock()
 
-	var ProcessSystemSummaryWg sync.WaitGroup
+	aggregateSummaryMap(tempSummarizerMap)
```

Comment:
Do we really need to aggregate and store in the DB? IMO we should keep the raw data in DB and use agg before publishing it to SaaS? WDY @rajaSahil @seswarrajan

Reply:
It is not going to create any issues. And the agg is false by default in karmor summary right? That is why I suggested to keep it raw.